src/styles/_catalog.scss

```
@@ -1,9 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
```

``
```suggestion
  grid-template-columns: repeat(auto-fill, $cardWidth));
```
maybe its better